aura/strike_dataGrid/strike_dataGridHelper.js

```diff
@@ -91,15 +91,25 @@
 				component.set('v.showTable', true);
 			}
 		}
-		formattedData.rows.forEach(function(row){		
-			$A.createComponent(
-				"c:strike_row",
-				{
-					"fields": row.fields
-				},
-				createRowCallback
+
+		if(formattedData.rows > 0) {
```

Comment:
formattedData.rows is an array , it cannot be compared with an Integer, it would always return false. it has to be formattedData.rows.length >0 , and in the forEach loop it won't enter into that section as its already handled for length > 0 . The issue here is that when its empty its not creating an empty component.